idl/storage_service.idl.hh

```diff
@@ -55,6 +55,7 @@ struct raft_topology_snapshot {
     std::vector<canonical_mutation> topology_mutations;
     std::vector<canonical_mutation> cdc_generation_mutations;
     std::vector<canonical_mutation> topology_requests_mutations;
+    utils::chunked_vector<canonical_mutation> auth_mutations;
```

Comment:
Why auth mutations are part of topology?

Reply:
piggy-backing? a bit similar to cdc_generation_mutations. Do they deserve their own RPC?